src/providers/colors/ColorProvider.cpp

```diff
@@ -128,6 +128,9 @@ void ColorProvider::initTypeColorMap()
     initColor(ColorType::ThreadMessageHighlight,
               getSettings()->threadHighlightColor,
               HighlightPhrase::FALLBACK_THREAD_HIGHLIGHT_COLOR);
+
+    initColor(ColorType::AutomodHighlight, getSettings()->automodHighlightColor,
+              HighlightPhrase::FALLBACK_HIGHLIGHT_COLOR);
```

Comment:
This should probably have its own fallback color `#404040`, which was the default automod highlight color before

Reply:
Added `HighlightPhrase::FALLBACK_AUTOMOD_HIGHLIGHT_COLOR` with color `QColor(64, 64, 64);`